src/propsheet/propsheet.vcxproj.filters

```diff
@@ -45,9 +45,6 @@
     <ClCompile Include="PropSheetHandler.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="readLink.cpp">

```

Comment:
Just curious: why don't we need this anymore? (I don't even know what readLink is)

Reply:
It looks like it's a file that doesn't exist anymore and the .filters file didn't get updated until I changed the Visual Studio Project file for other reasons.

This is typically because @zadjii-msft was working on something outside Visual Studio and didn't update the filters file at some point in the past. :P